backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/select/internal/BpkSelectImpl.kt

```diff
@@ -72,16 +72,27 @@ internal fun BpkSelectImpl(
         )
         ExposedDropdownMenu(
             expanded = if (status != BpkFieldStatus.Disabled) expanded else false,
-            modifier = Modifier.background(BpkTheme.colors.surfaceDefault).fillMaxWidth(),
+            modifier = Modifier
+                .background(BpkTheme.colors.surfaceDefault)
+                .fillMaxWidth(),
             onDismissRequest = { expanded = false },
         ) {
             options.forEachIndexed { index, option ->
-                val itemBackgroundColor = if (index == selectedIndex) BpkTheme.colors.surfaceHighlight else BpkTheme.colors.surfaceDefault
+                val itemBackgroundColor =
+                    if (index == selectedIndex) BpkTheme.colors.surfaceHighlight else BpkTheme.colors.surfaceDefault
                 DropdownMenuItem(
                     modifier = Modifier
                         .height(BpkSpacing.Lg.times(2))
                         .background(itemBackgroundColor),
-                    text = { BpkText(text = option, color = BpkTheme.colors.textPrimary, maxLines = 1, overflow = TextOverflow.Ellipsis) },
+                    text = {
+                        BpkText(
+                            text = option,
+                            color = BpkTheme.colors.textPrimary,
+                            maxLines = 1,
+                            overflow = TextOverflow.Ellipsis,
+                            style = BpkTheme.typography.label1,
```

Comment:
this is the actual fix - two things happened here: we didn't set a style and because somewhere in the dropdown component the default style is set we weren't actually using a backpack style, but also it was incorrect and should be `label1`